tst/EnergyPlus/unit/PurchasedAirManager.unit.cc

```diff
@@ -303,6 +303,11 @@ TEST_F(EnergyPlusFixture, IdealLoadsAirSystem_GetInput)
     EXPECT_ENUM_EQ(PurchAir(1).CoolingLimit, LimitType::NoLimit);
     EXPECT_ENUM_EQ(PurchAir(1).DehumidCtrlType, HumControl::ConstantSupplyHumidityRatio);
     EXPECT_ENUM_EQ(PurchAir(1).HumidCtrlType, HumControl::ConstantSupplyHumidityRatio);
+    // confirm a constant fuel efficiency defalt value of 1.0
+    EXPECT_EQ(PurchAir(1).heatFuelEffSched, Sched::GetScheduleAlwaysOn(*state));
+    EXPECT_EQ(PurchAir(1).coolFuelEffSched, Sched::GetScheduleAlwaysOn(*state));
+    EXPECT_EQ(PurchAir(1).heatFuelEffSched->getCurrentVal(), 1.0);
+    EXPECT_EQ(PurchAir(1).coolFuelEffSched->getCurrentVal(), 1.0);
```

Comment:
This unit test verifies that the default fuel efficiency values are set to `1.0` if the new fields are blank.